app/commands/user_track/acquired_trophy/create.rb

```diff
@@ -0,0 +1,41 @@
+class UserTrack::AcquiredTrophy::Create
+  include Mandate
+
+  initialize_with :user, :track, :category, :slug, send_email: true
+
+  def call
+    # Check to see if it exists already before doing any other expensive things
+    acquired_trophy = UserTrack::AcquiredTrophy.find_by(user:, track:, trophy:)
+    return acquired_trophy if acquired_trophy
+
+    # Check if the trophy should be awarded. Raise an exception if not
+    raise TrophyCriteriaNotFulfilledError unless trophy.award?(user, track)
+
+    # Build the trophy
+    begin
+      UserTrack::AcquiredTrophy.create!(
+        user:,
+        track:,
+        trophy:
+      ).tap do |user_track_acquired_trophy|
+        if trophy.send_email_on_acquisition? && send_email
+          User::Notification::CreateEmailOnly.(user, :acquired_trophy,
```

Comment:
I think we want full notifications, not email only. So we'll also want correct keys in the `locales` etc.